benchmark/messages.cpp

```diff
@@ -0,0 +1,111 @@
+// SPDX-FileCopyrightText: Copyright (c) 2024 Cisco Systems
+// SPDX-License-Identifier: BSD-2-Clause
+
+#include <quicr/detail/uintvar.h>
+
+#include <benchmark/benchmark.h>
+
+#include <quicr/detail/messages.h>
+#include <quicr/detail/serializer.h>
+
+static void
+Moq_EncodeGroupHeader(benchmark::State& state)
+{
+    for ([[maybe_unused]] const auto& _ : state) {
+        quicr::Serializer buffer;
+        auto hdr = quicr::messages::MoqStreamHeaderGroup{};
+        hdr.subscribe_id = 0x100;
+        hdr.track_alias = 0x100;
+        hdr.priority = 0xA;
+        hdr.group_id = 0x1;
```

Comment:
Better to move this out of the loop so it doesn't get measured. Then you don't need to do the ClobberMemory thing.

Reply:
Unless the intent is to see how bad the performance gets if you create these on the fly. In which case it would be good to have both tests: 1 where the buffer and message exist before the for loop, and one where they are also measured, and then we can compare.